ifsbench/launcher/bashlauncher.py

```diff
@@ -0,0 +1,75 @@
+# (C) Copyright 2020- ECMWF.
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+# In applying this licence, ECMWF does not waive the privileges and immunities
+# granted to it by virtue of its status as an intergovernmental organisation
+# nor does it submit to any jurisdiction.
+
+import datetime
+from pathlib import Path
+
+from ifsbench.launcher.launcher import Launcher, LaunchData
+
+class BashLauncher(Launcher):
```

Comment:
Conceptually, I'd argue it would be better to make this a `ScriptLauncher` or `ShellScriptLauncher` and let the fact that this uses Bash be an implementation detail.